### ISSUE TITLE
Exception in thread "main" java.lang.NullPointerException

### ISSUE THREAD

**Issue by devurandom** [iNPUTmice/caas#41]
When trying to run 3f1e91420340a7a404474412342e0e107363d4f3 I see following error:
```
$ env JAVA_HOME=/usr/lib/jvm/java-11 mvn package -pl caas-annotations,caas-app 
$ env JAVA_HOME=/usr/lib/jvm/java-11 /usr/lib/jvm/java-11/bin/java -jar caas-app/target/caas-app.jar test-user test-password  
Server is running Prosody 0.10.3
Exception in thread "main" java.lang.NullPointerException
        at java.base/java.util.TreeMap.compare(TreeMap.java:1291)
        at java.base/java.util.TreeMap.put(TreeMap.java:536)
        at java.base/java.util.TreeSet.add(TreeSet.java:255)
        at java.base/java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.TreeMap$KeySpliterator.forEachRemaining(TreeMap.java:2739)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at rocks.xmpp.extensions.disco.model.info.InfoDiscovery.getFeatures(InfoDiscovery.java:133)
        at im.conversations.compliance.xmpp.tests.AbstractDiscoTest.run(AbstractDiscoTest.java:32)
        at im.conversations.compliance.xmpp.TestExecutor.executeTestsFor(TestExecutor.java:52)
        at im.conversations.compliance.CommandLineLauncher.main(CommandLineLauncher.java:73)
```

Distribution is Fedora 29, Java is:
```
$ /usr/lib/jvm/java-11/bin/java --version
openjdk 11.0.1 2018-10-16
OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.1+13, mixed mode, sharing)
```

**Comment by schemacs** [comment]
Fixed in [babbler 4f057ba45915da8c70b234db690777a16efd7bf5](https://bitbucket.org/sco0ter/babbler/commits/4f057ba45915da8c70b234db690777a16efd7bf5).

**Comment by iNPUTmice** [comment]
You can remove the unecessary mod_mam_archive from your server. That will probably work around the problem. 
@schemacs Have you tested if this is available in the 0.8.0-snapshot. From looking at the website it looks like it is only available in master branch.

**Comment by schemacs** [comment]
@iNPUTmice It's my fault, just tried the fix. YES, this fix is only on master branch now.

**Comment by devurandom** [comment]
@iNPUTmice Thanks, disabling `mod_mam_archive` worked around the issue.

**Comment by iNPUTmice** [comment]
Should be fixed by 2fff74d7bbf3690f1714cf4774448d79e8f45d4d
